pkg/schema/error.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package schema
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/k14s/ytt/pkg/filepos"
+	"github.com/k14s/ytt/pkg/yamlmeta"
+)
+
+type errorWithContent struct {
+	contents string
+	Position *filepos.Position
+}
+
+func (e errorWithContent) Error() string {
+	panic("do not use directly")
+}
+
+func (e *errorWithContent) SetContext(doc *yamlmeta.Document) error {
```

Comment:
Odd how in "SetContext" what is actually happening is the content that is being set. Maybe we stick with one of either content or context?

Reply:
Removed the function entirely :D